cadc-registry/README.md

```diff
@@ -0,0 +1,15 @@
+# cadc-registry
+client implementation of IVOA Registry specifications
+
+### LocalAuthority.properties
+The LocalAuthority.properties file specifies which local service is authoritative for various site-wide functions. The keys are standardID values for the functions and the values are resourceID values for the service that implements that standard feature.
+
+Example:
+```
+ivo://ivoa.net/std/GMS#search-0.1 = ivo://cadc.nrc.ca/gms           
```

Comment:
Now that GMS is an IVOA recommendation, this should be: ivo://ivoa.net/std/GMS#search-1.0